### DISCUSSION TITLE
「蒙奇·D·路飞	meng qi d lu fei」这个词语引发了超级简拼不能使用的问题

### DISCUSSION THREAD

**Discussion by iDvel** [rime/home#967] [General]
```
# 这样的词语导致了「d」「r」这个字母不能参与超级简拼，如「daod」不会显示「到达」，「wenr」不会显示「温柔」。
蒙奇·D·路飞	meng qi d lu fei
乔治·R·R·马丁	qiao zhi r r ma ding

# 必须去掉空格才可以 ↓ 但是去掉空格后，不能使用「mqdlf」来打出「蒙奇·D·路飞」了。
蒙奇·D·路飞	mengqidlufei
```

请问这是一个 Bug，还是因为一些其他原因不能这样设置词库，有无什么办法？

**Comment by lotem** [comment]
第一個例子有個問題：
拼音輸入法要用拼音音節爲詞條編碼，輸入法程序才能夠根據方案中設定的規則識別簡拼。
按照以上假定，本程序認爲示例中的碼表定義了兩個單字母的新音節。
其在簡拼輸入中的行爲可以參照拼音`a`處理，如輸入`ayi`解釋爲全拼`a'yi`而不匹配`an'yi, ao'yi`等。

第二個例子屬於經典的碼表輸入法。編碼是一串字符，對輸入法程序而言他沒有結構。Rime按照輸入方案的定義實現輸入法，若無對輸入碼`mqdlf`的定義，預期他會打出一個編碼完全不同的詞是毫無道理的。

在拼音輸入方案裏，如果輸入能解析爲全拼，則不當簡拼解釋。
你需要在碼表裏區分拼音與非拼音編碼，從而解決部分編碼與簡拼混淆的問題。

**Comment by lotem** [comment]
關聯過往對相似問題的討論：
https://github.com/rime/librime/issues/72
https://github.com/rime/librime/issues/135
https://github.com/rime/squirrel/issues/319
https://github.com/rime/squirrel/issues/442
https://github.com/rime/weasel/issues/582
